src/planscape/planscape/settings.py

```diff
@@ -199,6 +199,9 @@
         "dj_rest_auth.jwt_auth.JWTCookieAuthentication",
     ),
     "NON_FIELD_ERRORS_KEY": "global",
+    "DEFAULT_FILTER_BACKENDS": ["django_filters.rest_framework.DjangoFilterBackend"],
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
```

Comment:
I'm always a fan of limit/offset, but our UX designs look like they might be more compatible with a Page based approach, so not sure if this is what we want

Reply:
True. Okay, I'll switch it back. I prefer limit/offset, anyway.